project.ipynb

```diff
@@ -0,0 +1,329 @@
+{
```

Comment:
<p>Would be good to get a description of the dataset here, and what you notice about it on first glance. </p><p>Some things I would include would be:</p><ul><li>num features</li><li>num examples</li><li>proportion of examples in test vs train set</li><li>is this a regression or classification problem? or something in between?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI-Core-Assignments/project-1-42FraMeWork/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>